src/components/MaanedsbeloepAvansertBeregning/__tests__/hooks.test.tsx

```diff
@@ -0,0 +1,277 @@
+import { IntlProvider } from 'react-intl'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+
+import { getSelectedLanguage } from '@/context/LanguageProvider/utils'
+// Import the mocked functions for controlling their behavior
+import { useAppSelector } from '@/state/hooks'
+// Import actual selectors to reference them in our mocks
```

Comment:
Tror ikke vi trenger kommentarer på importer ;)
```suggestion
import { useAppSelector } from '@/state/hooks'
```